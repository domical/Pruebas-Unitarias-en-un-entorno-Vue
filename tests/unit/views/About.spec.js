import { shallowMount } from '@vue/test-utils';
import About from '@/views/About.vue';

describe('About.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(About);
    expect(wrapper.exists()).toBe(true);
  });

  it('matches snapshot', () => {
    const wrapper = shallowMount(About);
    expect(wrapper.html()).toMatchSnapshot();
  });
});