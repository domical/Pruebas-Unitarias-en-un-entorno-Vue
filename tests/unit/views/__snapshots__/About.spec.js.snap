// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.vue matches snapshot 1`] = `
<div>
  <h1>About Us</h1>
  <p>This is the About page.</p>
</div>
`;

exports[`AboutView.vue Genera un snapshot de la vista AboutView 1`] = `
<div>
  <h2>About</h2>
  <p>Definitivamente esto es el about</p>
</div>
`;
